themes/58649066-2b6f-4a5b-af6d-c3d21d16fc00/preferences.json

```diff
@@ -1,15 +1,92 @@
-{
-    "uc.private-browsing-top-bar.solid-bg": "Replaces the gradient on the toolbar with a solid color.", 
-    "uc.private-browsing-top-bar.no-bg": "Removes the background color on the toolbar entirely.", 
-    "uc.private-browsing-top-bar.no-border": "Removes the colored border that surrounds the page.",
-    "uc.private-browsing-top-bar.use-zen-theme-color": "Use the Zen theme color as the highlight color.",
-    "uc.private-browsing-top-bar.color-blue": "Use a blue color as the highlight color. Overrides any ticked colors above this setting.",
-    "uc.private-browsing-top-bar.color-teal": "Use a teal color as the highlight color. Overrides any ticked colors above this setting.",
-    "uc.private-browsing-top-bar.color-green": "Use a green color as the highlight color. Overrides any ticked colors above this setting.",
-    "uc.private-browsing-top-bar.color-yellow": "Use a yellow color as the highlight color. Overrides any ticked colors above this setting.",
-    "uc.private-browsing-top-bar.color-orange": "Use a orange color as the highlight color. Overrides any ticked colors above this setting.",
-    "uc.private-browsing-top-bar.color-brown": "Use a brown color as the highlight color. Overrides any ticked colors above this setting.",
-    "uc.private-browsing-top-bar.color-red": "Use a red color as the highlight color. Overrides any ticked colors above this setting.",
-    "uc.private-browsing-top-bar.color-pink": "Use a pink color as the highlight color. Overrides any ticked colors above this setting.",
-    "uc.private-browsing-top-bar.hide-icon": "Hides the private browsing icon."
-}
+[
+    {
+        "property": "uc.private-browsing-top-bar.highlighting-style",
+        "label": "Highlighting Style:",
+        "type": "dropdown",
+        "options": [
+            {
+                "label": "Gradient",
+                "value": "gradient"
+            },
+            {
+                "label": "Solid",
+                "value": "solid"
+            },
+            {
+                "label": "None",
+                "value": "nonestyle"
```

Comment:
This is not needed as we automatically add a type none :)

Reply:
I did see that, but from my tests, it seems like 'none' is the default when a Zen mode is first installed? I mainly just added a separate 'none' value so that the initial default value _does_ show something and so that users know the mod is working. If there's a way to mark a default/initial value in the dropdown list and I just missed it, then I'll remove the none value and use that.

Reply:
Actually, yes! Yesterday I merged this pr https://github.com/zen-browser/components/blob/a4ffded16b2461012e63c3bb12e250091739ee93/src/ZenThemesImporter.mjs#L128 that allows theme authors to add default values

Reply:
This will be available next release though!

Reply:
Excellent news! I'll update my PR now to include the defaultValue property. Is there a way to add a label for the 'none' property as currently it's just a hyphen?

Reply:
Currently there's no way, but I can figure out something :)